### PR TITLE
feat(message-system): introduce easy way to test local jws

### PR DESCRIPTION
**Pull request by vytick**
Add simple option to test message system with local jws

### CODE REVIEW

**Review by matejkriz — APPROVED** [review]
Great, thanks. I was thinking about environment variable for this (same as we have for analytics logs EXPO_PUBLIC_IS_ANALYTICS_LOGGER_ENABLED), but I think this is better to see it in the git diff so you are not so confused why you can't fetch new remote version...

What about to skip sequence number check too? 
```javascript
if (currentSequence < config.sequence || FORCE_LOCAL_JWS /* Allow to skip sequence check for local testing */) { 
```

File: suite-common/message-system/src/messageSystemThunks.ts [hunk 1]
```diff
@@ -22,7 +22,18 @@ import {
 } from './messageSystemSelectors';
 import { jws as configJwsLocal } from '../files/config.v1';
 
+// this is for local development purpose
+// set to true to always fetch local jws
```
Comment (matejkriz) [inline suite-common/message-system/src/messageSystemThunks.ts]: Nitpick, feel free to ignore
```suggestion
// Enable this for local development purposes:
// set to true to always fetch local JWS
```

Comment (vytick) [reply suite-common/message-system/src/messageSystemThunks.ts]: Great idea: 88529b1dbd

**Pull request merged**